plugins/broker-redis/webpack.config.js

```diff
@@ -1,13 +1,13 @@
 const path = require("path");
 const webpack = require("webpack");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
-const nodeExternals = require("webpack-node-externals");
 
 /** @type {import("webpack").Configuration} */
 module.exports = {
   mode: "production",
   target: "node",
   entry: "./src/index.ts",
+  externals: ["@amplication/code-gen-utils", "@amplication/code-gen-types"],
```

Comment:
What does it do?

Reply:
it won't bundle these dependencies in the code as they are already installed in DSG